### PR TITLE
Check if image exists before getting url

### LINKED ISSUE #1497: Error when running release report
when trying to run a release report, it fails due to  backend error, there could be a related JS issue as well:

Image

### PR DESCRIPTION
**Pull request by brianjp93**
- fixes #1497

### CODE REVIEW

File: templates/admin/release_report_detail.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h1>Boost</h1>
         </div>
         <div class="flex flex-col h-full gap-y-4">
 
-          {% if version.release_report_cover_image.url %}
+          {% if version.release_report_cover_image %}
```
Comment (GregKaleka) [inline templates/admin/release_report_detail.html:60]: Safer? Overkill?

```suggestion
          {% if version.release_report_cover_image and version.release_report_cover_image.url %}
```

Comment (brianjp93) [reply templates/admin/release_report_detail.html:60]: Fine with me

Comment (GregKaleka) [reply templates/admin/release_report_detail.html]: I don't think it's possible to have a non-null `FileField` instance and have it return a falsey `url`, so probably overkill.

**Review by GregKaleka — APPROVED** [review]
LGTM

**Pull request merged**